sql/_13_issues/_23_1h/cases/cbrd_24598.sql

```diff
@@ -0,0 +1,19 @@
+-- This test case verifies CBRD-24598 issue.
+-- Core occures because of not freed memory.
+-- Core occures in second execution query on all after 10.x version.
+-- Core does not occures in 9.x version.
+-- Allocated memory on the global heap must be freed on the global heap.
+-- After error fix both query should be executed normaly.
+
+drop table if exists t;
+
+-- prepare required 
+prepare q from 'select decode (?, '''', c, NULL, c, -1) from table ({1}) as t (c)'; 
+
+-- success
+execute q using 'A'; 
+
+-- success
+execute q using 1; 
```

Comment:
Please check the query below once more.
```
-- success
execute q using 'A';
```
The error code will be the answer.

I would like to be able to check whether this problem has been resolved through a test case fail.

Reply:
opposite case and repeated execution case is added.